### ISSUE TITLE
Places still return 404

### ISSUE THREAD

**Issue by bloedboemmel** [immich-app/immich#10223]
### The bug

Still after #10121 many places return a 404 code
If I click the places in the explore page (https://redacted.com/explore), I always get the correct location:
https://redacted.com/search?query=%7B%22city%22%3A%22Absberg%22%7D

But if I click on the same place on the places page (https://redacted.com/places/) it redirects me to https://redacted.com/places/photos/69433182-f878-4bf8-9ab7-521b22e063e4 with an Not Found - 404
undefined

### The OS that Immich Server is running on

Synology

### Version of Immich Server

v1.106.2

### Version of Immich Mobile App

v1.106.1

### Platform with the issue

- [X] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
name: immich

services:
  immich-server:
    container_name: immich_server
    image: ghcr.io/immich-app/immich-server:${IMMICH_VERSION:-release}
    volumes:
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
      - /etc/localtime:/etc/localtime:ro
      - /volume1/externalLibrary:/usr/src/app/external:ro
    env_file:
      - .env
    ports:
      - 2283:3001
    depends_on:
      - redis
      - database
    restart: always


  immich-machine-learning:
    container_name: immich_machine_learning
    # For hardware acceleration, add one of -[armnn, cuda, openvino] to the image tag.
    # Example tag: ${IMMICH_VERSION:-release}-cuda
    image: ghcr.io/immich-app/immich-machine-learning:${IMMICH_VERSION:-release}
    # extends: # uncomment this section for hardware acceleration - see https://immich.app/docs/features/ml-hardware-acceleration
    #   file: hwaccel.ml.yml
    #   service: cpu # set to one of [armnn, cuda, openvino, openvino-wsl] for accelerated inference - use the `-wsl` version for WSL2 where applicable
    volumes:
      - model-cache:/cache
    env_file:
      - .env
    restart: always

  redis:
    container_name: immich_redis
    image: registry.hub.docker.com/library/redis:6.2-alpine@sha256:84882e87b54734154586e5f8abd4dce69fe7311315e2fc6d67c29614c8de2672
    restart: always

  database:
    container_name: immich_postgres
    image: registry.hub.docker.com/tensorchord/pgvecto-rs:pg14-v0.2.0@sha256:90724186f0a3517cf6914295b5ab410db9ce23190a2d9d0b9dd6463e3fa298f0
    environment:
      POSTGRES_PASSWORD: ${DB_PASSWORD}
      POSTGRES_USER: ${DB_USERNAME}
      POSTGRES_DB: ${DB_DATABASE_NAME}
      POSTGRES_INITDB_ARGS: '--data-checksums'
    volumes:
      - ${DB_DATA_LOCATION}:/var/lib/postgresql/data
    restart: always
    command: ["postgres", "-c" ,"shared_preload_libraries=vectors.so", "-c", 'search_path="$$user", public, vectors', "-c", "logging_collector=on", "-c", "max_wal_size=2GB", "-c", "shared_buffers=512MB", "-c", "wal_compression=on"]

volumes:
  model-cache:
```

### Your .env content

```Shell
# You can find documentation for all the supported env variables at https://immich.app/docs/install/environment-variables

# The location where your uploaded files are stored
UPLOAD_LOCATION=./library
# The location where your database files are stored
DB_DATA_LOCATION=./postgres

# The Immich version to use. You can pin this to a specific version like "v1.71.0"
IMMICH_VERSION=release

# Connection secret for postgres. You should change it to a random password
DB_PASSWORD=something

# The values below this line do not need to be changed
###################################################################################
DB_USERNAME=something
DB_DATABASE_NAME=immich
```

### Reproduction steps

```bash
1. Click on Explore
2. Click on the first place. It should return the place
3. Go to Explore -> Places. Click on the first place. It should return 404
```

### Relevant log output

```shell
Nothing to see in the logs
```

**Comment by michelheusschen** [comment]
Only the `/explore` page was fixed initially, but #10142 also fixed the issue on the `/places` page and will be included in the next release

**Comment by bloedboemmel** [comment]
Ah I saw this but thought it's not connected to the issue. Thanks :)
